include/nbl/core/hash/blake.h

```diff
@@ -12,15 +12,19 @@ struct blake3_hash_t
 {
 	inline bool operator==(const blake3_hash_t&) const = default;
 
-	uint8_t data[BLAKE3_OUT_LEN];
+	uint8_t data[BLAKE3_OUT_LEN] = { {0} };
 };
 
+static_assert(sizeof(blake3_hash_t) == BLAKE3_OUT_LEN);
+
+// TODO, Arek: why not in source and limit to translation unit as part of Nabla library, again we make a wrapper just to expose 3rdparty to our public headers and with blake case we need to link it when in static mode anyway because it's not header only library
```

Comment:
ok actually those blake3 structs we use are built with basic types only & all in headers